en/setup/dev_env_linux_ubuntu.md

```diff
@@ -63,6 +63,11 @@ sudo add-apt-repository --remove ppa:team-gcc-arm-embedded/ppa
 
 <!-- NOTE: RaPi docker toolchain (for comparison) here: https://github.com/PX4/containers/blob/master/docker/Dockerfile_armhf -->
 
+> **Warning** Use Ubuntu **18.04** or [Docker](https://github.com/PX4/containers/blob/master/docker/Dockerfile_armhf) for building RasPi!
+  *Do not use Ubuntu 20.04*. The *armhf* or *aarch64* toolchains on a Ubuntu 20.04 host machine depend on *libc6-2.29* which is not available on RPi at time of writing (it has *libc6-2.27*).
+
+They can pass the build and provide binary files, but is extreme new so cannot run on actual Pi. The really working one is documented separately in user docs.
```

Comment:
Add link to the user doc?

Reply:
[This section](https://dev.px4.io/master/en/setup/building_px4.html#other-boards) is a little disorganized. Should all of those boards locate somewhere else?(for example, user docs).
I think only this roughly link should be provided here: https://docs.px4.io/master/en/flight_controller/

Reply:
@SalimTerryLi 
1. "The really working one is documented separately in user docs." like @bkueng says, where is that link to go to?
2. The information should be in one place. I would be happy for this section you point to go into the user docs. What we would leave here is a link to that. 

We should fix this now and do that other work as a separate action. OK?